### ISSUE TITLE
 Rename playful ignition

### ISSUE THREAD

**Issue by jack67889** [hanabi/hanabi.github.io#749]
Seems to be a pretty random and confusing name. Players are confused who should blind-play after calling it ‘ignition’. So I propose to make it clearer by changing the name to: Chop Move Pull (CM Pull).

**Comment by pianoblook** [comment]
The Ignition occurs from the immediate next player, which signals that the CM'd card is actually playable - hence, 'playful ignition'. It also doesn't always occur from a 'pull' situation, so that would be much more confusing

**Comment by jack67889** [comment]
As far as I can tell the chop move always pulls at the moment, could you clearify what you are saying there? See #747 for the rewrite I did.

I think the word 'ignition' is the main problem here. It's a pretty random word and everyone is used to just saying blind-play your finesse. Also, because for 'double ignitions' the blind-play can skip the next player, it's unclear to players if that also applies for this convention (which was not a question, but an observation). Furthermore, the definition of 'ignition' consists of two rules and the second one doesn't apply for this convention: "Nothing needs to be 'proved' to the player who got clued, which is unlike a Finesse, and unlike a Bluff" (because you need to prove the chop moved card is actually playable).

I'm a fan of using terminology players are familiar with, instead of coming up with new words, which makes conventions harder to remember imho.

**Comment by jack67889** [comment]
I guess you mean if someone does an (trash) order chop move or a tempo clue chop move on a playable card? Didn't really think pull would be confusing in that case (I would say the chop move still pulls). But maybe just 'Chop Move Play Clue' or something could also be an option.

**Comment by Eliclax** [comment]
Yeah I've had this issue before. I've even played with friends who took the "ignition" at face value, meaning the ignition is on the last person with a playable finesse position before the chop moved player. I'm sure this isn't how the convention was intended to work. I think "Chop Move Play Clue" is a better name, or "Chop Move Double Play", or actually what we really need is a name for finesse position signal shifts.

**Comment by timotree3** [comment]
There was an issue recently where I proposed that we define Ignitions such that they usually call for a blind-play from Bob. (#675) This change was made to the doc for Double Ignitions but not for the rest. (#732) If that change was in place, would it solve the confusion about who needs to blind-play from a Playful Ignition?

**Comment by jack67889** [comment]
Timo, I’m aware, but I really think it doesn’t help (also there is no “rest” of course).

Maybe I should take this in small steps: I would say calling the convention ‘Chop Move Ignition’ is already a huge improvement over the current name. For all of the above reasons. 

It could be that it changes over time, but it seems that trouble of defining ‘ignition’ for “just a normal finesse blind-play on the next player” is simply not worth it. In the past weeks I’ve heard already multiple times that that can’t be right.

**Comment by pianoblook** [comment]
Naming moves like this 'Ignitions' is a positive theoretical step, I think, but I'd also be down for trying to think of an even better classification. 

My understanding of what defines an Ignition is that it's any form of ['Call-Out Bluff'](https://github.com/hanabi/hanabi.github.io/blob/main/misc/bluff-types.md) that requires an immediate blind-play in order for some second card to properly receive its Play signal. This is already distinct from _Unnecessary_ style clues where the base Signal is immediately clear to the clued player.

I think there's definitely room for better definitional groupings, especially knowing that there will likely be more conventions in the future that utilize this sort of _Igniting _effect. For example, last year I was pushing for the expansion + unification of Ignitions with more Blaze-and-Shadow-style moves (see #365 and #449, (plus _Dumpster Fires_). 

A different approach could be to combine Playfuls with the kind of blind-play that occurs from Pestilent and Cursed moves (plus others I'm probably not remembering atm), while leaving Blaze-style moves - those that specifically require a positional match component - as a distinct concept.

Anyway, TLDR I'm totally be down with a name change. Zam suggested something like 'Playful' because it is descriptive of the desired outcome. "Chop Move Ignition" would be, too. 

Or if we want to cut the _Ignition_ part, we could consider something like "Cursed" or "Pestilent": Jinxed CM, Deceitful CM, Cheeky CM. Or lean into the _Kick_ component: Chop Kick, Kick Finesse, Kick Move, ...etc

**Comment by jack67889** [comment]
Thanks for your elaboration @pianoblook, I get the use of ignition is an improvement theoretical, wasn't sure if it's in practice. But you convinced me we should probably make it stick over time, to be future proof and that will probably work out.

> That's not quite correct. Playful Ignition is unrelated to the playability of the clued card, which is exactly what the second rule is. It is related to the playability of ancillary cards though.

I see what you are saying now, still depends a bit on how you actually chop move the card. For me the wording in the call-out bluff doc was a bit clearer (first time I saw that).

> Hopefully Piano has illustrated above how and why an Ignition is different than a Pull. 

Yes, I was trying to avoid using the word ignition, by not refering to the ignited card but the one being cm'ed (like the 5 double pull before).

> I'd be fine with renaming Playful Ignition to Chop Move Ignition.

Okay with me. For me, including Chop Move in the name to make it more descriptive, was the main point.

**Comment by pianoblook** [comment]
I'll second Chop Move Ignition. Playful is a cute name but doesn't seem like it adds much, especially if there's concern of it causing confusion/being harder to remember.

**Comment by Zamiell** [comment]
ok i renamed it to *Chop Move Ignition*
